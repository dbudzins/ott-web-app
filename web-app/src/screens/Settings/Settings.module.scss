@use '../../styles/variables';
@use '../../styles/theme';
