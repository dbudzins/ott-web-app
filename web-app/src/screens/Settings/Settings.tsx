import React from 'react';

import styles from './Settings.module.scss';

type Props = {
  prop?: string;
};

const Settings: React.FC<Props> = ({ prop }: Props) => {
  return (
    <div className={styles.Settings}>
      <p>{prop}</p>
    </div>
  );
};

export default Settings;
