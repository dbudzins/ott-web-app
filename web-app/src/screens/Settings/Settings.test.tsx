import React from 'react';
import { render } from '@testing-library/react';

import Settings from './Settings';

describe('<Settings>', () => {
  test('renders and matches snapshot', () => {
    const { container } = render(<Settings />);

    expect(container).toMatchSnapshot();
  });
});
