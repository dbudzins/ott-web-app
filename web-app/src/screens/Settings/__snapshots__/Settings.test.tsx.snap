// Vitest Snapshot v1

exports[`<Settings> > renders and matches snapshot 1`] = `
<div>
  <div>
    <p />
  </div>
</div>
`;
